src/EnergyEfficientAgent.cpp

```diff
@@ -223,14 +221,15 @@ namespace geopm
         for (size_t ctl_idx = 0; ctl_idx < (size_t) m_num_freq_ctl_domain; ++ctl_idx) {
             struct m_region_info_s current_region_info {
                 .hash = (uint64_t)m_platform_io.sample(m_signal_idx[M_SIGNAL_REGION_HASH][ctl_idx]),
-                .hint = (uint64_t)m_platform_io.sample(m_signal_idx[M_SIGNAL_REGION_HINT][ctl_idx]),
+                .hint = GEOPM_REGION_HINT_UNKNOWN,
```

Comment:
if the 'hint" field is not used, can we just remove it from the m_region_info_s?

Reply:
done